kubecost/Chart.yaml

```diff
@@ -10,6 +10,6 @@ annotations:
       url: https://www.kubecost.com
 dependencies:
   - name: finops-agent
-    version: "v0.2.2"
```

Comment:
Version needs to be verified when https://github.com/kubecost/finops-agent-chart/pull/21 goes in.

Reply:
let's merge this now and update the chart when the image is ready- right?